### PR TITLE
Feature/discovar denovo

### PR DESCRIPTION
**Pull request by jerowe**
* [ ] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Discovar Denovo

### CODE REVIEW

File: recipes/discovar-denovo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  number: 0
+  skip: True # [osx]
+
+package:
+  name: discovardenovo
+  version: '52488'
+
+requirements:
+  build:
+      - libgcc
```
Comment (bgruening) [inline recipes/discovar-denovo/meta.yaml]: libgcc is not needed here.

File: recipes/kmergenie/build.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -x -e
+
+export CC=${PREFIX}/bin/gcc
```
Comment (bgruening) [inline recipes/kmergenie/build.sh]: GCC :)

File: recipes/kmergenie/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+about:
+  home: http://kmergenie.bx.psu.edu/
+  license: ''
```
Comment (bgruening) [inline recipes/kmergenie/meta.yaml]: license missing

File: recipes/kmergenie/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+build:
+  number: 0
+
+package:
+  name: kmergenie
+  version: '1.7016'
+
+requirements:
+  build:
+      - gcc
```
Comment (bgruening) [inline recipes/kmergenie/meta.yaml]: - gcc # [linux]
- llvm # [osx]

File: recipes/kmergenie/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      - zlib
+      - r-base
+      - r-essentials
+      - python 2.7.12
```
Comment (bgruening) [inline recipes/kmergenie/meta.yaml]: please do not pin a specific python version, you can skip py3k if you want

Comment (jerowe) [inline recipes/kmergenie/meta.yaml]: how do i do that?

Comment (bgruening) [inline recipes/kmergenie/meta.yaml]: like here for example: https://github.com/bioconda/bioconda-recipes/blob/60d2ea750430dbe0191f6f8d5639f90fc1223bd9/recipes/pynast/1.2.2/meta.yaml#L12

**Comment by jerowe** [comment]
I'd like to do a setup.py for this instead of just throwing the whole thing in in $PREFIX/bin, but I'm really not familiar with python packages with c utilities.

I've been taking a look at pysam, and I'm trucking along, but I'd appreciate advise. Is there anyone on the team with experience in packaging python dists?

File: recipes/kmergenie/meta.yaml (hunk 5 above)
Comment (jerowe) [inline recipes/kmergenie/meta.yaml]: It should be python3 anyways. Somedays I should stop working much earlier. ;-)

**Comment by jerowe** [comment]
I moved kmergenie over to its branch.

Should I merge the discovar denovo recipe?

**Comment by bgruening** [comment]
@jerowe sure go on! :)

**Comment by jerowe** [comment]
Cool. Thanks. ;-)

**Pull request merged**